packages/analytics-js/src/services/ErrorHandler/ErrorHandler.ts

```diff
@@ -52,18 +69,18 @@ class ErrorHandler implements IErrorHandler {
       (error as Error).message = errorMessage;
     }
 
-    const normalisedError = isTypeOfError ? error : new Error(errorMessage);
+    const normalizeError = isTypeOfError ? error : new Error(errorMessage);
```

Comment:
normalizeError is ok or you would like normalizedError instead?

Reply:
It was a typo.